lib/openactive/dataset_site/settings.rb

```diff
@@ -94,6 +102,20 @@ def booking_service
         )
       end
 
+      def access_service
+        OpenActive::Models::WebAPI.new(
+          name: 'Open Booking API',
+          description: "API that allows for seamless booking experiences to be created for #{data_feed_descriptions.to_sentence} available from #{organisation_name}",
```

Comment:
`data_feed_descriptions.to_sentence` should also be converted to lower case

Reply:
Done